src/app/cheat-sheets/game-base/trains/trains.component.html

```diff
@@ -27,11 +21,11 @@
     </li>
     <li>[#locomotives in front]-[#wagons]-[#locomotives behind]</li>
   </ul>
-  Generally can use the following configurations for trains for an average game (remove the locos behind if not using terminal stations):
+  Example train consists for an average game (locos on both ends of the train):
   <ul>
-    <li>2-4-2 trains for ore</li>
-    <li>1-2-1 train for oil</li>
-    <li>1-1-1 train for acid (uranium mine)</li>
+    <li>2-4-2 trains, for ore.</li>
+    <li>1-2-1 trains, for oil.</li>
+    <li>1-1-1-1 trains, for uranium (1 cargo wagon for the uranium ore, 1 fluid wagon with sulfuric acid for the mining drills).</li>
```

Comment:
I think we want to recommend this consist?

- 1 locomotive
- 1 uranium ore wagon
- 1 sulfuric acid fluid wagon
- 1 locomotive (to speed up the train compared to just 1 loco)